teamwork/templates/adminlte/lib/_main_header.html

```diff
@@ -52,22 +52,31 @@
                 <li class="dropdown user user-menu">
                     <a href="#" class="dropdown-toggle" data-toggle="dropdown">
                         <img src="{% avatar_url size=50 %}" class="user-image" alt="User Image">
-                        {# <span class="hidden-xs">{% firstof request.user.get_short_name request.user.get_username %}</span> #}
-                        <span class="hidden-xs">Account</span>
+                        <span class="hidden-xs">{% firstof request.user.get_short_name request.user.get_username %}</span>
+                        {# <span class="hidden-xs">Account</span> #}
                     </a>
                     <ul class="dropdown-menu">
                         {% block user_header %}
+
                         <li class="user-header">
                             <img src="{% avatar_url size=180 %}" class="img-circle" alt="User Image">
                             <p>
                                 {% firstof request.user.get_short_name request.user.get_username %}
-                                <small>Grepper since {{ request.user.date_joined }}</small>
+                                <small>Grepper since {{ request.user.date_joined|date:"M Y" }}</small>
+                                <small>{% if request.user.profile.isGT %}
+                                Developer
+                                {% elif request.user.profile.isProf %}
+                                Professor
+                                {% else %}
+                                Student
+                                {% endif %}</small>
```

Comment:
Looks good, but instead of a title we display their location/school or nothing at all. Also, not all instructors are Professors, instead of Professor should be Instructor.

Reply:
This is only shown to the current user, so it's more to specify their account type rather than display information about them such as a title or location